packages/backend/src/scheduler/SchedulerClient.ts

```diff
@@ -264,19 +262,14 @@ export class SchedulerClient {
 
     async addScheduledDeliveryJob(
         date: Date,
-        scheduler: Scheduler | CreateSchedulerAndTargets,
+        scheduler: ScheduledDeliveryPayload,
         schedulerUuid: string | undefined,
     ) {
         const graphileClient = await this.graphileUtils;
 
-        const payload: ScheduledDeliveryPayload = schedulerUuid
```

Comment:
be sure this is correct.

Reply:
revert some logic here: https://github.com/lightdash/lightdash/pull/13924/commits/78188f93e8f04c05877cfe8bb4a45dbd8cf6c80b